From 13f2d053cb0b7d7d66aefb118052b89379845d14 Mon Sep 17 00:00:00 2001
From: LAMBS Pierre-Antoine <pierreantoine.lambs@gmail.com>
Date: Mon, 14 Aug 2023 15:23:50 +0200
Subject: [PATCH 2/3] bryton860: Use the correct MMC slot

Signed-off-by: LAMBS Pierre-Antoine <pierreantoine.lambs@gmail.com>
---
 include/configs/sunxi-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/sunxi-common.h b/include/configs/sunxi-common.h
index 7260eb72a4..925b9ca237 100644
--- a/include/configs/sunxi-common.h
+++ b/include/configs/sunxi-common.h
@@ -94,7 +94,7 @@
 
 /* mmc config */
 #ifdef CONFIG_MMC
-#define CONFIG_MMC_SUNXI_SLOT		0
+#define CONFIG_MMC_SUNXI_SLOT		2
 #endif
 
 #if defined(CONFIG_ENV_IS_IN_MMC)
-- 
2.43.0

